Problem 2.cpp

```diff
@@ -0,0 +1,21 @@
+//Time Complexity : O(n)
+// Space Complexity :O(1)
+// Did this code successfully run on Leetcode :Yes
+// Any problem you faced while coding this :No
```

Comment:
Shouldn't the space for Lowest Common Ancestors problems be O(height of tree) due to the call stack?